sources/assets/Stride.Core.Assets.Quantum/AssetCompositeHierarchyPropertyGraph.cs

```diff
@@ -624,6 +624,12 @@ private void UpdateAssetPartBases()
             if (part.Base is null)
                 continue;
 
+            if (part.Base.BasePartAsset is null)
```

Comment:
This line can be merged with the previous one.

``` csharp
if (part.base?.BasePartAsset is null)
```

Reply:
Done.